app/assets/stylesheets/users.scss

```diff
@@ -46,3 +49,15 @@ $red: #ff0000;
     margin: 0 0 30px;
   }
 }
+
+.col.l3.m5.s5.user-show {
+  height: 30em;
+  background-color: #a3cded;
```

Comment:
Color literals like `#a3cded` should only be used in variable declarations; they should be referred to via variable everywhere else.